crates/distribution-types/src/parsed_url.rs

```diff
@@ -71,13 +71,15 @@ pub struct ParsedArchiveUrl {
 impl TryFrom<&Url> for ParsedGitUrl {
     type Error = ParsedUrlError;
 
+    /// Supports url both with `git+` prefix and without. With prefix is PEP 508, without is
+    /// `tool.uv.sources`.
     fn try_from(url_in: &Url) -> Result<Self, Self::Error> {
         let subdirectory = get_subdirectory(url_in);
 
         let url = url_in
             .as_str()
             .strip_prefix("git+")
-            .ok_or_else(|| ParsedUrlError::MissingUrlPrefix(url_in.clone()))?;
```

Comment:
Is `MissingUrlPrefix` still used?

Reply:
No, it's already removed: https://github.com/astral-sh/uv/pull/3263/files#diff-1a2282604ae94f9266d311e67fd6e0c365c2a76a7ef0fbd707bfc19e6dde48e5L18-L19